client/pages/results.jsx

```diff
@@ -7,12 +7,18 @@ export default class Results extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
+      isSaved: false,
+      isError: false,
       route: parseRoute(window.location.hash),
       inputValue: null,
-      results: null
+      results: null,
+      info: null
     };
     this.handleDescription = this.handleDescription.bind(this);
     this.handleAuthor = this.handleAuthor.bind(this);
+    this.handleSave = this.handleSave.bind(this);
+    this.handleHeading = this.handleHeading.bind(this);
+    this.handleSavedItem = this.handleSavedItem.bind(this);
```

Comment:
Only methods that are passed as a callback to another component or function need to be bound.